modules/reference/pages/command/featureUtility-installServerFeatures.adoc

```diff
@@ -24,6 +24,13 @@ Install the server features for the `myserver` server:
 featureUtility installServerFeatures myserver
 ----
 
+Install the features for the `myserver` server, which includes custom user features:
+
+----
```

Comment:
for both the usage examples- italicize runtime-vesrion and remove the brackets:

featureUtility installServerFeatures myserver --featuresBom=my.user.features:myBOM:_runtime-version_

where  _runtime-version_ is the version of the Open Liberty server where you want to install the feature, for example, 21.0.0.11.

you'll have to add the [subs=+quotes] format to thew codeblocks to get them to render properly